package.json

```diff
@@ -20,7 +20,19 @@
         "async"
     ],
     "version": "1.3.2",
-    "main": "./src/index.ts",
+    "main": "./dist/fp-toolkit.cjs",
+    "module": "./dist/fp-toolkit.mjs",
+    "types": "./dist/fp-toolkit.d.ts",
+    "exports": {
+        ".": {
+            "require": "./dist/fp-toolkit.cjs",
+            "import": "./dist/fp-toolkit.mjs",
+            "types": "./dist/fp-toolkit.d.ts"
+        }
+    },
+    "files": [
+        "dist"
+    ],
```

Comment:
This makes it so we're only including `package.json`, `dist/*`, `LICENSE.md`, and `README.md` in our package.

Reply:
😻